<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="pspmovie.qrc" >:/images/sonypsp.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QPushButton" name="transcodeButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="contextMenuPolicy" >
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="toolTip" >
     <string>Transcode movie to PSP format</string>
    </property>
    <property name="statusTip" >
     <string>Transcode movie to PSP format</string>
    </property>
    <property name="icon" >
     <iconset resource="pspmovie.qrc" >:/mainwin/images/folder_video.png</iconset>
    </property>
    <property name="iconSize" >
     <size>
      <width>96</width>
      <height>96</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="xferButton" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>0</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="contextMenuPolicy" >
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="toolTip" >
     <string>Exchange movies between PSP and PC</string>
    </property>
    <property name="statusTip" >
     <string>Exchange movies between PSP and PC</string>
    </property>
    <property name="icon" >
     <iconset resource="pspmovie.qrc" >:/mainwin/images/reload.png</iconset>
    </property>
    <property name="iconSize" >
     <size>
      <width>96</width>
      <height>96</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>130</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title" >
     <string>Progress</string>
    </property>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>50</y>
       <width>75</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text" >
      <string>Frames :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>160</x>
       <y>50</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text" >
      <string>of total</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberFrames" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>50</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="numDigits" >
      <number>7</number>
     </property>
     <property name="segmentStyle" >
      <enum>QLCDNumber::Filled</enum>
     </property>
     <property name="value" stdset="0" >
      <double>900000</double>
     </property>
     <property name="intValue" stdset="0" >
      <number>900000</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberFramesTotal" >
     <property name="geometry" >
      <rect>
       <x>210</x>
       <y>50</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="numDigits" >
      <number>7</number>
     </property>
     <property name="segmentStyle" >
      <enum>QLCDNumber::Filled</enum>
     </property>
     <property name="value" stdset="0" >
      <double>900000</double>
     </property>
     <property name="intValue" stdset="0" >
      <number>900000</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>20</y>
       <width>251</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value" >
      <number>24</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="action_About" />
   </widget>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionTranscode" />
    <addaction name="actionTransfer" />
    <addaction name="separator" />
    <addaction name="actionE_xit" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionTranscode" >
   <property name="text" >
    <string>&amp;Transcode</string>
   </property>
  </action>
  <action name="actionTransfer" >
   <property name="text" >
    <string>T&amp;ransfer</string>
   </property>
  </action>
  <action name="action_About" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionE_xit" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="pspmovie.qrc" />
 </resources>
 <connections/>
</ui>
